map_model/src/make/turns.rs

```diff
@@ -1,11 +1,14 @@
-use std::collections::{BTreeMap, HashMap, HashSet};
+use std::collections::{BTreeMap, HashSet};
 
 use anyhow::Result;
 use lyon::geom::{CubicBezierSegment, Point, QuadraticBezierSegment};
 
-use geom::{Angle, PolyLine, Pt2D};
+use geom::{PolyLine, Pt2D};
 
-use crate::{Intersection, Lane, LaneID, LaneType, Map, RoadID, Turn, TurnID, TurnType};
+use crate::{
+    map::turn_type_from_road_geom, Intersection, Lane, LaneID, LaneType, Map, RoadID, Turn, TurnID,
```

Comment:
This might be naitve, but this _feels_ wrong. Previously everything else is imported is a `impl`/`struct`, but here I'm importing in a function, which seems inconsistant. I don't know if this matters though.